cmd/livepeer/starter/starter.go

```diff
@@ -1412,3 +1420,37 @@ func getBroadcasterPrices(broadcasterPrices string) []BroadcasterPrice {
 
 	return pricesSet.Prices
 }
+
+type KeystorePath struct {
```

Comment:
```suggestion
type keystorePath struct {
```

I think we don't need it to be accessible outside this package.